cmd/start.go

```diff
@@ -443,3 +449,23 @@ func editConfigFile() error {
 	}()
 	return configmanager.SaveFromFile(tmpFile)
 }
+
+func start(app *app.App, conf *config.Config) error {
+	startResult := (*app).Start(*conf)
+	if startResult == nil && startCmdArgs.Flags.Foreground {
+		defer awaitForInterruption(app)
+	}
+	return startResult
+}
```

Comment:
```suggestion
func start(app app.App, conf config.Config) error {
	if err := app.Start(conf); err != nil {
		return err
	}
	if startCmdArgs.Flags.Foreground {
		awaitForInterruption(app)
	}
	return nil
}
```

Reply:
Is it ok to pass by value?

Reply:
It has no effect because it is an interface, which points to an actual value.